### ISSUE TITLE
Letter J missing when writing text

### ISSUE THREAD

**Issue by zaddok** [signintech/gopdf#201]
When using the font `EB Garamond` from google, the lowercase letter j does not render properly. Only the dot is printed. This is super strange. Not sure if I am doing something wrong, or if there is a special trick for some fonts.  Just asking here first incase there are any obvious known issues I need to look out for before I waste a few hours trying to debug it 😀 

```
package main

import (
	"github.com/signintech/gopdf"
	"log"
)

func main() {
	pdf := gopdf.GoPdf{}
	pdf.Start(gopdf.Config{PageSize: *gopdf.PageSizeA4})
	pdf.AddPage()

        // Downloaded from https://fonts.google.com/specimen/EB+Garamond
	err := pdf.AddTTFFont("EBGaramond", "EBGaramond-Regular.ttf")
	if err != nil {
		log.Print(err.Error())
		return
	}

	err = pdf.SetFont("EBGaramond", "", 14)
	if err != nil {
		log.Print(err.Error())
		return
	}

	pdf.SetX(30)
	pdf.SetY(40)
	pdf.Text("hijk")

	pdf.WritePdf("test.pdf")
}
```

**Comment by zaddok** [comment]
Here is a sample demonstrating the letter Ἰ (with accent) not working.

```
package main

import (
	"github.com/signintech/gopdf"
	"log"
)

func main() {
	pdf := gopdf.GoPdf{}
	pdf.Start(gopdf.Config{PageSize: *gopdf.PageSizeA4})
	pdf.AddPage()

        // Downloaded from https://fonts.google.com/specimen/EB+Garamond
	err := pdf.AddTTFFont("EBGaramond", "EBGaramond-Regular.ttf")
	if err != nil {
		log.Print(err.Error())
		return
	}

	err = pdf.SetFont("EBGaramond", "", 14)
	if err != nil {
		log.Print(err.Error())
		return
	}

	pdf.SetX(30)
	pdf.SetY(40)
	pdf.Text("hijk Ἰ")

	pdf.WritePdf("test.pdf")
}
```

**Comment by oneplus1000** [comment]
I have tested from the code you provided, but I found that the characters are not missing (I tested on version v0.10.5)

<img width="936" alt="Screen Shot 2564-12-05 at 12 47 39" src="https://user-images.githubusercontent.com/1155104/144735563-aadf90fb-f73d-42f9-b49a-60bac32d435e.png">

<img width="472" alt="Screen Shot 2564-12-05 at 12 47 54" src="https://user-images.githubusercontent.com/1155104/144735565-6cbd0e56-db03-47da-8ce4-9b126a29970a.png">

**Comment by zaddok** [comment]
I wish I kept the original folder on my computer which I originally created the test code that was intended to reproduce the problem. When I was having the problem with "j" in this font, I switched to an alternate source for the font file. Then, later on, the problem with Ἱ appeared (while still using that alternate source). Now I switch back to the Google version, and everything is now working.  

I just wound back from 0.10.5 to 0.10.4 and 0.10.3 and it now all works fine with the Google font file. I am not sure what has changed, but whatever has changed, it is all working for me. If I encounter the problem again I will create a zip file with the code and the comment so it is properly reproducible.  Thank you for this library it really is great.
